packages/common/src/utils/media.ts

```diff
@@ -49,3 +51,37 @@ export const getLegacySeriesPlaylistIdFromEpisodeTags = (item: PlaylistItem | un
 
 export const isLiveChannel = (item: PlaylistItem): item is RequiredProperties<PlaylistItem, 'contentType' | 'liveChannelsId'> =>
   item.contentType?.toLowerCase() === CONTENT_TYPE.liveChannel && !!item.liveChannelsId;
+
+export const createVideoMetaArray = (media: PlaylistItem, episodesLabel?: string) => {
```

Comment:
I think you can change these function names (also both functions below this one) to something like `createVideoMetadata`, `createPlaylistMetadata`, I don't think you have to specify in the name that it's an array

Reply:
You are right. I went along with the old name (`formatVideoMetaString`). But this is better.

Reply:
Changed!